src/screens/Home/index.tsx

```diff
@@ -10,7 +10,7 @@ const HomeScreen = (): JSX.Element => {
     <div className="app">
       <header className="app-header">
         <img src={logo} className="app-logo" alt="logo" />
-        <p>{t('sample_page.message', { codeSample: '<code>src/App.tsx</code>' })}</p>
+        <p data-test-id="home-message">{t('sample_page.message', { codeSample: '<code>src/App.tsx</code>' })}</p>
```

Comment:
We use `camelCase` for ID and `kebab-case` for class -> https://nimblehq.co/compass/development/code-conventions/html/#ids-and-class-names
`data-test-id` is also the ID, so we should change it as well 🙏 
```suggestion
        <p data-test-id="homeMessage">{t('sample_page.message', { codeSample: '<code>src/App.tsx</code>' })}</p>
```